docsSrc/How_Tos/Write_an_item.md

```diff
@@ -3,71 +3,80 @@
 ## Write a simple item
 
 ```fsharp
-open System
-open Amazon.DynamoDBv2.Model
-open DynamoDb.Ok
-
-type SimpleObject =
-    { Id: Guid
-      Email: String
-      IsVerified: Boolean
-      DateOfBirth: DateTime
-      Balance: Decimal }
-
-let client = new AmazonDynamoDBClient()
-
-let write (o: SimpleObject): Async<Result<unit, DynamoDbError list>> =
-    [ Attr("Id", ScalarGuid o.Id)
-      Attr("Email", ScalarString o.Email)
-      Attr("Verified", ScalarBool o.IsVerified)
-      Attr("DateOfBirth", ScalarDate o.DateOfBirth)
-      Attr("Balance", ScalarDecimal o.Balance) ]
-    |> Write.putItem client "CustomersTable"
+module Simple =
```

Comment:
Had to move this into a module since `write` is defined twice and that's not allowed in F#.